test/library/packages/UnitTest/UnitTest.chpl

```diff
@@ -1081,18 +1081,14 @@ module UnitTest {
     var testCount = 0;
     var _tests: [1..0] argType;
     
-    proc addTest(test) lifetime this < test {
-      // var tempTest = new Test();
-      // param test_name = test: string;
-      // if !canResolve(test_name,tempTest) then
-      //   compilerError(test + " is not callable");
+    pragma "unsafe"
+    proc addTest(test) {
       this._tests.push_back(test);
       this.testCount += 1;
     }
 
-    proc addTests(tests) lifetime this < tests {
-      /*if isString(tests) then
-        compilerError("tests must be an iterable, not a string");*/
+    pragma "unsafe"
```

Comment:
I think we should add a comment explaining why we have these.

Reply:
Okay Sure.

Reply:
I have updated it. Thanks